dart/common/Subscriber.cpp

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2014-2015, Georgia Tech Research Corporation
```

Comment:
Change the copyright year to 2015. The beginning year should be the year of the code was initially created, and the ending year should be the year of the release of DART.